third-party/linshare/src/test/java/org/apache/james/linshare/client/TechnicalAccountDTO.java

```diff
@@ -0,0 +1,116 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.linshare.client;
+
+import java.util.List;
+
+import org.testcontainers.shaded.com.google.common.collect.ImmutableList;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class TechnicalAccountDTO {
+
+    private final String uuid;
+    private final long creationDate;
```

Comment:
I'm still not ok with this, we have rich data types in Java, why should we live with a `long` which is clearly not a date type?

Reply:
also, it looks like most of the fields here are never read. Why would we want to describe a DTO with field we don't use? We can very well only map fields that we care about, right?

Reply:
Right, i thought that before and now i agree with you.In this case, we just need only **uuid**.